Cargo.toml

```diff
@@ -18,7 +18,7 @@ log = "0.4.17"
 once_cell = "1.16.0"
 serde = { version = "1.0.151", features = [ "derive" ] }
 serde_json = "1.0.91"
-sqlx = { version = "0.6.2", features = [ "runtime-tokio-rustls", "postgres", "macros", "migrate", "time", "json", "bigdecimal", "offline" ] }
```

Comment:
Was the offline feature removed or why is it gone?

Reply:
ah I see, it is gone, so those sqlx files in the folder are the new way?

Reply:
Yeah, so I've spent couple of hours battling with a compilation error because I had newer version of `cargo sqlx`, which saved queries in `.sqlx` folder, but our code tried to read older `sqlx-data.json`. So we should either mention in readme that you need to install older version of sqlx or to just update our sqlx crate dependency (which I did here). 

And yeah, offline feature is removed, now it's default behavior, and instead of single `sqlx-data.json` file there's a `.sqlx` folder with each query in separate file